recipes/dogstatsd-ruby.rb

```diff
@@ -1,8 +1,8 @@
 #
-# Cookbook Name:: datadog
+# Cookbook:: datadog
 # Recipe:: dogstatsd-ruby
 #
-# Copyright 2013-2105, Datadog
+# Copyright:: 2013-2105, Datadog
```

Comment:
While I am happy about the optimism in thinking that this Chef's cookbook will be used for ~80 more years I think we should be more humble 😄

```suggestion
# Copyright:: 2013-2015, Datadog
```
I don't know if we should maybe bump everything to 2020 (in a different PR)?

Reply:
It's a bit of a hassle to update the copyright every year in all repos, and I'm not sure how important it is from a legal point of view. We should ask.

Reply:
👍 OpenTelemetry for example removed it a while back so maybe it's not needed